src/lib.rs

```diff
@@ -1000,7 +1002,8 @@ impl<Fs: FileSystem + Default> ResolverGeneric<Fs> {
             tracing::trace!(tsconfig = ?tsconfig, "load_tsconfig");
 
             // Extend tsconfig
-            if let Some(tsconfig_extend_specifier) = &tsconfig.extends {
+            // TODO multiple extends
+            if let Some(ExtendsField::Single(tsconfig_extend_specifier)) = &tsconfig.data.extends {
```

Comment:
TS `extends` supports arrays, which oxc does not support yet. We should fix in another PR.

Furthermore, the crate has an impl for resolving extends. Not sure if we want to use it or not: https://docs.rs/typescript_tsconfig_json/0.1.1/typescript_tsconfig_json/struct.TsConfigJson.html#method.resolve_extends_chain

Reply:
@Boshen Yeah it's buried in the release notes: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#supporting-multiple-configuration-files-in-extends

Reply:
lolz, I'll have to revert one of my previous change to this.

Reply:
https://github.com/oxc-project/oxc-resolver/issues/101